exercises/ex_atm.py

```diff
@@ -0,0 +1,164 @@
+import sys
+
+FILE_PATH_INDEX = 1
+END_ATM_SERVICE = "-1"
+PASSWORD_KEY = "password"
+BALANCE_KEY = "balance"
+
+
+def store_file_content(file_path):
+    """Read file and move atm information to dictionary.
+
+    The lines of the file will contain user's id, balance and password,
+    with comma's separating them.
+
+    Args:
+        file_path (str): path to atm information file.
+
+    Returns:
+        dict. with user's id as keys,
+            and user's password and balance in another dict as values.
```

Comment:
When one dictionary (or list or anything else) is in another the term for that is "nested." Your comment is therefore better expressed as "in a nested dict as values."

Its clearer what you mean this way